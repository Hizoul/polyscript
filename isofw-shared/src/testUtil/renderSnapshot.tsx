import * as React from "react"
const render = require("preact-render-to-json")

const renderSnapshot = (component: any, msg: string) => {
  expect(render(component)).toMatchSnapshot(msg)
}

export default renderSnapshot
