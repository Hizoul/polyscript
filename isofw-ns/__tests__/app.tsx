import NativeContent from "isofw-ns/app/start"
import renderSnapshot from "isofw-shared/src/testUtil/renderSnapshot"
import * as React from "react"

test("Verify app rendering", () => {
  renderSnapshot(<NativeContent />, "rendered")

})