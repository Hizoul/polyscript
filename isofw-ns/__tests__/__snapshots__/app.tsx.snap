// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify app rendering: rendered 1`] = `
<page>
  <actionBar
    text="isofw"
  />
  <scrollView>
    <stackLayout>
      <button
        text="isofwbtn"
      />
      <button
        text="shared"
      />
      <progress
        value={50}
      />
    </stackLayout>
  </scrollView>
</page>
`;
